src/stamp/preprocessing/extractor/conchv1_5.py

```diff
@@ -0,0 +1,20 @@
+try:
+    from transformers import AutoModel 
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "conchv1_5 dependencies not installed."
+        " Please reinstall stamp using `pip install 'stamp[conchv1_5]'`"
```

Comment:
Please add the relevant dependencies in the toml file, otherwise this won't work: https://github.com/EzicStar/STAMP/blob/0b4760cdecd71e04cad5667d450ccbab70bbbec2/pyproject.toml#L47-L72